src/OrchardCore.Modules/OrchardCore.Forms/Models/SelectPart.cs

```diff
@@ -5,7 +5,16 @@ namespace OrchardCore.Forms.Models
     public class SelectPart : ContentPart
     {
         public SelectOption[] Options { get; set; }
+
         public string DefaultValue { get; set; }
+
+        public SelectEditorOption Editor { get; set; }
+
+        public LabelOptions LabelOption { get; set; }
+
+        public string Label { get; set; }
+
+        public ValidationOptions ValidationOption { get; set; }
```

Comment:
same as above 
Similar to `FormElementPart` and `FormInputElementPart` let's have this moved new part `FormElementLabelPart` and provide views for it.

Let's keep `InputPart` and `SelectPart` unchanged and  for label add `FormElementLabelPart` to content type Input and Select using migration